README.md

```diff
@@ -153,13 +152,14 @@ install_deps()
 
 ### Importing `pip` dependencies with `pip_import` (legacy)
 
-The deprecated `pip_import` can still be used if needed.
+The deprecated `pip_import` can still be used if needed. It is the only packaging rule that supports Python 2,
+which has been [sunsetted since January 1st, 2020](https://www.python.org/doc/sunset-python-2/). 
 
 ```
 load("@rules_python//python/legacy_pip_import:pip.bzl", "pip_import", "pip_repositories")
 
 # Create a central repo that knows about the dependencies needed for requirements.txt.
-pip_import(   # or pip3_import
```

Comment:
Let's not even encourage Python 3 usage with `pip_import`, because I don't want to field maintenance questions about it.